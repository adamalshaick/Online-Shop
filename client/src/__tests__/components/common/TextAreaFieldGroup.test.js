import React from "react";
import TextAreaFieldGroup from "../../../components/common/TextAreaFieldGroup";
import renderer from "react-test-renderer";

const wrapper = renderer.create(<TextAreaFieldGroup />).toJSON();

describe("render component", () => {
  it("renders component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
