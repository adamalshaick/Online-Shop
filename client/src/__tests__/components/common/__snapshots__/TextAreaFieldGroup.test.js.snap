// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component renders component 1`] = `
<div
  className="form-group"
>
  <textarea
    className="form-control form-control-lg"
    style={
      Object {
        "height": "200px",
      }
    }
  />
</div>
`;
